### PR TITLE
[edw] e2e test for `/pipelines`

### PR DESCRIPTION
**Pull request by edmondwinston**
# Description

This PR adds automatic login to the original test object. It also add a create-and-delete test case for the `/pipelines` page.

# Checklist
- [x] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [x] I have performed a self-review of my own code
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] If new documentation has been added, relative paths have been added to the appropriate section of `docs/mint.json`

cc: @johnson-mage

### CODE REVIEW

**Comment by edmondwinston** [comment]
@johnson-mage

Need a hand mate - I'm not sure why the test failed in CI with the admin creds.

**Comment by johnson-mage** [comment]
> @johnson-mage
> 
> Need a hand mate - I'm not sure why the test failed in CI with the admin creds.

Got it, I'll take a look when I get a chance. Thanks @edmondwinston

File: mage_ai/frontend/playwright.config.ci.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ export default defineConfig({
     cwd: '../../',
     env: {
       PYTHONPATH: '.',
+      REQUIRE_USER_AUTHENTICATION: '1',
```
Comment (johnson-mage) [inline mage_ai/frontend/playwright.config.ci.ts:47]: @edmondwinston So the issue with the login credentials failing was that we needed to enable user authentication.

**Comment by johnson-mage** [comment]
@edmondwinston Thanks for doing all the heavy lifting with this PR, but it looks like it's in a good place now. I made a bunch of revisions cuz even after adding user authentication to the CI/CD workflow, there were a bunch of issues I ran into causing the tests to fail (e.g. After logging in, it only redirects you to the Overview page if there's at least 1 pipeline run. Otherwise, it redirects to the Pipelines page. There were also some issues with the UI interactions happening too quickly before the page could load, or if it's a brand new project, certain UI elements are not present until you enable certain features in the project settings.).

**Review by johnson-mage — APPROVED** [review]

**Pull request merged**